superset/connectors/druid/models.py

```diff
@@ -71,6 +71,7 @@ def _fetch_metadata_for(datasource):
 
 class JavascriptPostAggregator(Postaggregator):
     def __init__(self, name, field_names, function):
+        Postaggregator.__init__(self, function, field_names, name)
```

Comment:
Edits to this file seems unrelated to your PR title

Reply:
I agree. The thing is that I started on the issue but gave a title related to the first I had started tackling. Do you suggest changing the title or split the PR into a few ones? (I'd favor the first.)